### ISSUE TITLE
Mapping labels to class IDs

### ISSUE THREAD

**Issue by sakethbachu** [f1recracker/pytorch-deeplab-v3-plus#1]
Hi, I was wondering if you have mapped the RGB values of the labels to the class ids as required by the cross entropy loss.
